doc/tutorials/Chapter_2/g_constant.dart

```diff
@@ -0,0 +1,31 @@
+/// Copyright (C) 2023 Intel Corporation
```

Comment:
There doesn't seem to be any references to this file in the text. Perhaps it is worth mentioning it, as it is done for other files?